packages/core/onKeyDown/index.md

```diff
@@ -0,0 +1,24 @@
+---
+category: Sensors
+---
+
+# onKeyDown
```

Comment:
Instead of creating folders for each of them, I actually think we can just put them all under `onKeyStroke` (multiple named exports).

Sorry, I didn't make that clear, I can handle that if you want. Thank you

Reply:
My first thought was this, but then I took them apart again.😂